OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,201 @@
-How to build the image
-======================
-Before you Start
-----------------
-1. **FIRST TIME ONLY:**
-   - Update the `OracleSOASuite/setenv.sh` file to point
-     to the right locations for the following environment variables
-     specific to your environment (See top of file):    
-     `DC_USERHOME`, `DC_REGISTRY_SOA`, `DC_REGISTRY_DB`    
-
-SOA/BPM/OSB domain images
--------------------------
-1. You must have the install binaries downloaded from the
-   [Oracle Technology Network](http://www.oracle.com/technetwork/middleware/soasuite/downloads/index.html) site before proceeding. 
-2. From the current directory run these commands. You will 
-   be prompted with all the information. Carefully review 
-   it and Confirm to proceed. 
-
-       # Use BUILD_OPTS to add extra arguments to the docker build command
-       sh buildDockerImage.sh -v 12.2.1.2
+SOA on Docker
+=============
+
+Sample Docker configurations to facilitate installation, configuration, and environment setup for Docker users. This project includes quick start dockerfiles for SOA 12.2.1.2.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle WebLogic Infrastructure 12.2.1.2.0.
+
+At the end of this configuration there will be 3 containers running : 1) DB Container 2) WLS Admin Server Container 3) WLS Managed Server (SOA Server) Container.
+The containers will be connected using a Docker User Defined network 
+
+
+## Pre-Requisite
+
+
+## Create a User Defined network
+
+
+In this configuration creation of a user defined network will enable the communication between the containers just using container names. User defined network option was preferred over the container linking option as the latter is now deprecated.
```

Comment:
You can remove the second sentence from this paragraph as container linking has been deprecated for a while now.